addons/pos_mercado_pago/models/mercado_pago_pos_request.py

```diff
@@ -0,0 +1,33 @@
+import logging
+import requests
+
+_logger = logging.getLogger(__name__)
+
+
+REQUEST_TIMEOUT = 10
+
+class MercadoPagoPosRequest:
+    def __init__(self, mp_bearer_token):
+        self.mercado_pago_bearer_token = mp_bearer_token
+
+    def _mercado_pago_get_endpoint(self):
+        return 'https://api.mercadopago.com'
+
+    def _call_mercado_pago(self, method, endpoint, payload):
+        """ Make a request to Mercado Pago POS API.
+        :param method: "GET", "POST", ...
+        :param endpoint: The endpoint to be reached by the request.
+        :param payload: The payload of the request.
+        :return The JSON-formatted content of the response.
+        """
+        endpoint = f"{self._mercado_pago_get_endpoint()}{endpoint}"
```

Comment:
```suggestion
        endpoint = MERCADO_PAGO_API_ENDPOINT + '/' + endpoint
```
with the current implementation, you don’t prevent domain injection (if `endpoint` is another domain)

Reply:
Done
(
even if I don't understand the difference here:
endpoint = MERCADO_PAGO_API_ENDPOINT + '/' + endpoint
endpoint = f"{MERCADO_PAGO_API_ENDPOINT}{endpoint}
)

Reply:
@papyDoctor if `endpoint` is something like `foo:bar@baz.com`, you will make a request to baz.com